fare/file_management/templates/file_management/create.html

```diff
@@ -27,6 +27,7 @@ <h2>Create record</h2>
             </div>
             <div class="form-group {{ 'has-error' if form.contributor_name.errors }}">
               <label for="contributor_name">{{ form.contributor_name.label }}</label>
+              <small>(separare i diversi autori con una virgola)</small>
```

Comment:
Here, and in all the other fronted strings, we need to understand what's the best policy for translations in order to stick with them in all the rest of the platfom.
I mean, should we write in English and then translate or not?

Reply:
I think that write in English and then translate is the best way, i'll make the chages